SKA-ScienceDataProcessor/integration-prototype
# Initial Healthcheck test in app ./sip/execution_control/master_contro…

…ller



## Description:

There is a healthcheck funtion in the master controller folder and a test script but has been disabled for now. Would need to review it and work on it a bit to make it avaliable. 

Docker compose file in deploy/demos_23_11_18 directory has been updated to run healthcheck for master_controller. It will check wether the process (PID 1) is running every 30s.

## Testing instructions:


To quite bit hard to test this. 
You can manually check if the process is running by manually logging into the container using 
docker exec and then check by running "ps -opid" and see if pid 1 is running or not.

The instance to be rebuilt and tested is the execution_control/master_controller.
## Types of changes


## Checklist:
